_shared/project/README.md

```diff
@@ -0,0 +1,263 @@
+<a href="{{ cookiecutter.__github_url }}/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-BSD--2--Clause-success"></a>
+<a href="https://github.com/hypothesis/cookiecutters/tree/main/{{ cookiecutter._directory }}"><img src="https://img.shields.io/badge/cookiecutter-{{ cookiecutter._directory|replace("-", "--") }}-success"></a>
+<a href="https://black.readthedocs.io/en/stable/"><img src="https://img.shields.io/badge/code%20style-black-000000"></a>
+
+# {{ cookiecutter.name }}
+
+{{ cookiecutter.short_description }}
+
+{{- include("README/head.md") }}
+
+{%- if cookiecutter.get("console_script") == "yes" %}
+
+## Installing
+
+We recommend using [pipx](https://pypa.github.io/pipx/) to install
+{{ cookiecutter.name }}.
+First [install pipx](https://pypa.github.io/pipx/#install-pipx) then run:
+
+```terminal
+pipx install {{ cookiecutter.slug }}
+```
+
+You now have {{ cookiecutter.name }} installed! For some help run:
+
+```
+{{ cookiecutter.__entry_point }} --help
+```
+
+## Upgrading
+
+To upgrade to the latest version run:
+
+```terminal
+pipx upgrade {{ cookiecutter.slug }}
+```
+
+To see what version you have run:
+
+```terminal
+{{ cookiecutter.__entry_point }} --version
+```
+
+## Uninstalling
+
+To uninstall run:
+
+```
+pipx uninstall {{ cookiecutter.slug }}
+```
+{%- endif %}
+
+## Setting up Your {{ cookiecutter.name }} Development Environment
+
+First you'll need to install:
+
+* [Git](https://git-scm.com/).
+  On Ubuntu: `sudo apt install git`, on macOS: `brew install git`.
+* [GNU Make](https://www.gnu.org/software/make/).
+  This is probably already installed, run `make --version` to check.
+* [pyenv](https://github.com/pyenv/pyenv).
+  Follow the instructions in pyenv's README to install it.
+  The **Homebrew** method works best on macOS.
+  The **Basic GitHub Checkout** method works best on Ubuntu.
+  You _don't_ need to set up pyenv's shell integration ("shims"), you can
+  [use pyenv without shims](https://github.com/pyenv/pyenv#using-pyenv-without-shims).
+{%- if cookiecutter.get("services") == "yes" %}
+* [Docker](https://docs.docker.com/install/).
+  Follow the [instructions on the Docker website](https://docs.docker.com/install/)
+  to install it.  
+  You **don't** need to install Docker Compose, the development environment
+  will install it automatically for you in tox.  
+  You **do** need to set up the `docker` command to work without `sudo`,
+  on Linux this means following Docker's [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/).
+{%- endif %}
+{%- if cookiecutter.get("frontend") == "yes" %}
+* [Node](https://nodejs.org/) and npm.
+  On Ubuntu: `sudo snap install --classic node`.
+  On macOS: `brew install node`.
+* [Yarn](https://yarnpkg.com/): `sudo npm install -g yarn`.
+{%- endif %}
+{{- include("hacking/prerequisites") }}
```

Comment:
Is this `hacking` any more?

Reply:
This happened when I renamed `HACKING.md` to `README.md`: I didn't want to rename the include file as then I'd have to rename that file in every project that uses it. We can do that but I think it's a separate PR (this include is already on `main`, this PR just moves it into the shared dir).